fcrepo-api-x-model/src/main/java/org/fcrepo/apix/model/Ontology.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.fcrepo.apix.model;
+
+/**
+ * Marker interface for ontologies.
+ * <p>
+ * Contains various namespaces and values of significant ontologies.
+ * </p>
+ */
+public interface Ontology {
+
+    public static final String APIX_NS = "http://example.org/apix#";
```

Comment:
Yes, Let's add that to the agenda for today's meeting.  You raised a good point earlier about `http://fedora.info/definitions`, which would probably require buy-in from the community at large (outside of API-X), but makes a lot of sense;